docs/ExperimentConfig.md

```diff
@@ -160,6 +164,14 @@ machineList:
     __useAnnotation__ means whether you use annotation to analysis your code and generate search space. 
 	   
 	    Note: if you set useAnnotation=True, you should not set searchSpacePath.
+
+* __nniManagerIp__
+  * Description
+   
+    __nniManagerIp__ set the IP address of your machine which the nni manager process runs on. This field is an optional choice, if you don't set nniManagerIp, nni will use the IP of etho device.
+
+        Note: if you don't have eth0 device in your machine, we suggest you to set nniManagerIp manually.
```

Comment:
run ``ifconfig`` on NNI manager's machine to check if eth0 device exists. If not, we recommend to set nnimanagerIp explicitly.

Reply:
updated to new format.